packages/expr/package.json

```diff
@@ -12,17 +12,19 @@
     },
     "sideEffects": false,
     "dependencies": {
-        "eval-estree-expression": "^2.0.3",
+        "eval-estree-expression": "github:jonschlinkert/eval-estree-expression#9cf28d2",
```

Comment:
Need to use a git dependency until @SamyPesse fix https://github.com/jonschlinkert/eval-estree-expression/pull/12 is published.